Wrappers/Python/test/test_AstraOperators.py

```diff
@@ -106,4 +111,195 @@ def test_norm_simple3D_gpu(self):
         print ("norm A3", n)
         self.assertTrue(True)
         self.assertAlmostEqual(n, self.norm, places=2)
-    
\ No newline at end of file
+
+class TestAstraFlexible(unittest.TestCase):
+    def setUp(self): 
+        # Define image geometry.
+        N = 128
+
+        ig = ImageGeometry(voxel_num_x = N, voxel_num_y = N, 
+                        voxel_size_x = 0.1,
+                        voxel_size_y = 0.1)
+        
+        detectors = N
+        angles = np.linspace(0, np.pi, 180, dtype='float32')
+        ag = AcquisitionGeometry(geom_type='parallel',
```

Comment:
better use the new way of creating geometries, especially as it'll give warnings

Reply:
fixed